prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -48,9 +48,9 @@ elements:
   - name: filler_order_id
     mapper: use(message_id)
 
-# Removed until SimpleReport sends this column
-#  - name: test_result_status
-#    csvFields: [{name: Test_result_status}]
+  # Optional until SimpleReport team provides this.
+  - name: test_result_status
+    csvFields: [{name: Test_result_status}]
 
```

Comment:
This field is NOT being sent by simple report yet, but I'm using this as a test case for our new ability to add fields as optional on our end, so that simple report can implement at their leisure.